usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/LoadAllEntityIntoMemory.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.usecase;
+
+import org.mobilitydata.gtfsvalidator.domain.entity.ParsedEntity;
+
+/**
+ * Use case to load into memory GTFS entities. This use case is called after parsing a row from a GTFS .txt file.
+ */
+public class LoadAllEntityIntoMemory {
```

Comment:
This use case is a switch container. After discussion, we decided to implement the switch directly in main and remove this class.